exercises/practice/parallel-letter-frequency/parallel_letter_frequency_test.go

```diff
@@ -12,32 +12,38 @@ import (
 // Make concurrent calls to Frequency and combine results to obtain the answer.
 
 var (
-	euro = `Freude schöner Götterfunken
-Tochter aus Elysium,
-Wir betreten feuertrunken,
-Himmlische, dein Heiligtum!
-Deine Zauber binden wieder
-Was die Mode streng geteilt;
-Alle Menschen werden Brüder,
-Wo dein sanfter Flügel weilt.`
+	ralph_ellison = `I am an invisible man. No, I am not a spook like those who haunted Edgar Allan Poe;
+nor am I one of your Hollywood-movie ectoplasms. I am a man of substance, of flesh and bone,
+fiber and liquids—and I might even be said to possess a mind. I am invisible, understand,
+simply because people refuse to see me. Like the bodiless heads you see sometimes in circus sideshows,
+it is as though I have been surrounded by mirrors of hard, distorting glass.
+When they approach me they see only my surroundings, themselves, or figments of their imagination—indeed,
```

Comment:
```suggestion
When they approach me they see only my surroundings, themselves, or figments of their imagination - indeed,
```

Same as the previous comment.